curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -0,0 +1,269 @@
+---
+id: 66db529d37ad966480ebb633
+title: Build a Lunch Picker Program
+challengeType: 14
+dashedName: build-a-lunch-picker-program
+---
+
+# --description--
+
+In this lab, you'll build a program that helps in managing lunch options. You'll work with an array of lunches, add and remove items from the array, and randomly select a lunch option.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should create a `lunches` variable and assign it an empty array that will be used to store lunch items.
+
+2. You should create a function `addLunchToEnd` that takes a string parameter and adds it to the end of the `lunches` array and returns a string in the format:
+     
+   - `"[Lunch Item] added to the end of the lunch menu."`
+
+3. You should create a function `addLunchToStart` that takes a string parameter and adds it to the start of the `lunches` array and returns a string in the format:  
+   
+   - `"[Lunch Item] added to the start of the lunch menu."`
+
+4. You should create a function `removeLastLunch` that removes the last lunch item from the `lunches` array and:
+   
+   - If successful, returns a string in the format:  
+     - `"[Lunch Item] removed from the lunch menu (last)."`
```

Comment:
```suggestion
     - `"[Lunch Item] removed from the end of the lunch menu."`
```

This sounded a little off to me - but not really an issue to leave it, so no need to make the change if you don't want. The test would have to change as well.